src/libraries/System.Globalization.Extensions/tests/Normalization/StringNormalizationTests.cs

```diff
@@ -21,6 +21,11 @@ public void IsNormalized(string value, NormalizationForm normalizationForm, bool
             {
                 Assert.Equal(expected, value.IsNormalized());
             }
+            if (PlatformDetection.IsBrowser && (normalizationForm == NormalizationForm.FormKC || normalizationForm == NormalizationForm.FormKD))
+            {
+                // Browser's ICU doesn't support FormKC and FormKD
+                return;
```

Comment:
If you want to follow what we do in other places so that this is shown as Skipped in the test output, change this method to be `[ConditionalTheory]` and then instead of returning, `throw new SkipTestException("Browser's ICU...");`